### DISCUSSION TITLE
Date picker and Date saving printing problems

### DISCUSSION THREAD

**Discussion by sghriyadh** [Budibase/budibase#10573] [Help] [answered]
Hi

I checked forum and despite discussions there was no clear option

I want to collect date without time and print same date without time.

I made a coulmn "Echo Date"  Type is Date/Time. i selected "ignore time Zone"

![PXL_20230513_045702882](https://github.com/Budibase/budibase/assets/130876667/8096c924-3cae-417a-bd64-8bdf7738c052)

Then i made a form which uses Date picker without timezone but when i go to Data i see Date+time (time is zero/ignored).

![PXL_20230513_045720997](https://github.com/Budibase/budibase/assets/130876667/a88eb829-ff74-404f-9c98-38275040284c)

This picker is still sending Date without time(i suppose) as time is Zero 

![PXL_20230513_045626252](https://github.com/Budibase/budibase/assets/130876667/a664315a-42b6-4529-b569-b1ec6300aaec)

Anyway i went to make a screen to print the same date using repeater formula. but it shows Datewithtime (23-05-11T00:00:00.00Z
![PXL_20230513_045745528](https://github.com/Budibase/budibase/assets/130876667/ca5b7632-d8ad-43ba-9194-6228d22df86b)

This is what is printed finally . (still shows time)

![16839567646371428819223391447426](https://github.com/Budibase/budibase/assets/130876667/bcd0bba2-82ff-4799-8f30-0b520588e87b)

is there any way to use a repeater formula to hide the time and show only date ?

Can anyone write this repeater formula here for me please?

**Comment by diogenesbrussels** [comment, marked as answer]
Hey @sghriyadh,

Here is some quick help, you can use handlebars to format your date:
```
Current datetime : {{ now }}
Format datetime : {{date input "DD/MM/YYYY - HH:MM"}}
Format date : {{date input "DD/MM/YYYY"}}
```

In your case, something like:
{{date Repeater.Members.Echo Date "DD/MM/YYYY"}}

I hope this helps you!

**Comment by sghriyadh** [comment]
Thank you. finally i can relax about date print out.
